.gitlab-ci.yml

```diff
@@ -54,6 +54,36 @@ tox:
     - !reference [.load_github_key, script]
     - tox -e py
 
+# used by release
+.python_versions:
+  parallel:
+    matrix:
+      - BUILD_PY_VERSION: [python3.6]
+      - RELEASE_PY_VERSION: [python3.6]
+
+release:
+  before_script:
+    - git fetch --unshallow
+
+get_version:
+  stage: test
+  tags:
+    - dind
+  image: ${BASE_CONTAINER_REGISTRY}/python3.8-builder:${BASE_CONTAINER_VERSION}
```

Comment:
builder 3.8?

Reply:
This is just to get the version of indexd. Builder  version should not matter for the result.